core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1627,9 +1595,9 @@ void deleteReaderFiles(TranslogReader reader) {
             reader.path().resolveSibling(getCommitCheckpointFileName(reader.getGeneration())));
     }
 
-    void closeFilesIfNoPendingViews() throws IOException {
+    void closeFilesIfNoPendingRetentionLocks() throws IOException {
         try (ReleasableLock ignored = writeLock.acquire()) {
-            if (closed.get() && deletionPolicy.pendingViewsCount() == 0) {
+            if (closed.get() && deletionPolicy.pendingTranslogRefCount() == 0) {
                 logger.trace("closing files. translog is closed and there are no pending views");
```

Comment:
Remove mention of views from this trace message: `closing files; translog is closed and there are no pending retention locks`